### PR TITLE
ipip

### PR DESCRIPTION
**Pull request by 1265578519**
[sakura.zip](https://github.com/adolfintel/speedtest/files/1756808/sakura.zip)

### CODE REVIEW

**Comment by 1265578519** [comment]
This is a faster localization DAT, for reference, remote query is not recommended desirable.

**Comment by adolfintel** [comment]
I took a look at your sakura.zip file. Are you sure that this database can be included in this project? It has a copyright by Beijing Tentex Technology Co., Ltd., but I was unable to find more info about it because the site is in chinese.

**Comment by 1265578519** [comment]
I am sure I can use it, it is free, is no copyright issue and there are paid services, paid services more accurate dat file, and contains the county, IDC and other information.
https://www.ipip.net/api.html
https://myip.ipip.net/
https://www.ipip.net/ip.html
https://github.com/17mon/php
I'm a deprecated API because of the latency issue using free dat.
In the near future he will introduce English documents, I am in their QQ group.
Currently in the free ip database. cz88.net more comprehensive information, but I still more support ipip.net.

**Comment by dlewis23** [comment]
MaxMind is still a much more accurate IP database, With IPIP the free fails on IPv6 and whats there for v4 outside of China isn't very accurate at all. MaxMind should be used on the server so no latency issue.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,6 @@
+FROM php:5.6-apache
```
Comment (Angristan) [inline Dockerfile]: Any reason no to use `php:7.2-apache`?

**Comment by adolfintel** [comment]
@Angristan as far as I know, none, but I usually test this on 5.6 since it's the one installed on my server

**Comment by Angristan** [comment]
@adolfintel Ok, FYI speedtest works well with 7.2 :)

**Pull request closed without merging**